mucommander-core/src/main/java/com/mucommander/search/SearchDialog.java

```diff
@@ -55,14 +57,21 @@
 import com.mucommander.text.Translator;
 import com.mucommander.ui.action.ActionProperties;
 import com.mucommander.ui.action.impl.FindAction;
+import com.mucommander.ui.dialog.InformationDialog;
 import com.mucommander.ui.main.MainFrame;
+import com.mucommander.ui.text.*;
```

Comment:
formatting - we don't use wildcards in `import` statements

Reply:
done